servant-server/servant-server.cabal

```diff
@@ -48,6 +48,7 @@ library
       , attoparsec         >= 0.12 && < 0.14
       , bytestring         >= 0.10 && < 0.11
       , containers         >= 0.5  && < 0.6
+      , http-api-data
```

Comment:
Can we put an upper bound? Our general policy so far has been to do that, since we get conveniently notified by stackage maintainers when they're too restrictive.